docs/references/errors.md

```diff
@@ -110,7 +110,7 @@ The provided subscription is invalid. Subscriptions should have:
 * A `sink` field in the format of `service://<SERVICE_NAME>/<METHOD_NAME>`. When registering, service and method should have been previously registered as well.
 * Additional constraints may apply depending on the sink service contract
 
-Please look at the [Kafka documentation page](https://docs.restate.dev/develop/sdk/kafka) for more details on subscriptions and event handlers.
```

Comment:
you need to update this in the restate repo

Reply:
will do